src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -1418,17 +1432,6 @@ template <class TagList>
 using get_items =
     tmpl::filter<TagList, tmpl::not_<tmpl::bind<db::is_compute_tag, tmpl::_1>>>;
 
-namespace detail {
-template <class ItemsList, class ComputeItemsList>
-struct compute_dbox_type;
-
-template <class... ItemsPack, class ComputeItemsList>
-struct compute_dbox_type<tmpl::list<ItemsPack...>, ComputeItemsList> {
-  using type = decltype(db::create<tmpl::list<ItemsPack...>, ComputeItemsList>(
-      std::declval<item_type<ItemsPack>>()...));
-};
-}  // namespace detail
```

Comment:
Should the public facing alias below be moved up too and then get used in `db::create`?

Reply:
Ok, I will do that, and rearrange commits.  I plan to rebase this on #2714.

Reply:
ping on this? Line 1435/1438 just below `using compute_databox_type`

Reply:
Yeah, I accidentally mis-squashed that commit away before; fixed now.